ledger/catchpointtracker.go

```diff
@@ -1036,7 +1036,7 @@ func (ct *catchpointTracker) accountsUpdateBalances(accountsDeltas compactAccoun
 	if ct.log.GetTelemetryEnabled() {
 		root, rootErr := ct.balancesTrie.RootHash()
 		if rootErr != nil {
-			ct.log.Infof("accountsUpdateBalances: error retrieving balances trie root: %v", rootErr)
+			ct.log.Errorf("accountsUpdateBalances: error retrieving balances trie root: %v", rootErr)
```

Comment:
@cce I agree it's not _as serious_.  If you feel strongly, I can open a new PR to revert the log level change - let me know?

I felt a failure here likely implies an issue with how the software is written rather than how it's used.